x-pack/plugins/security_solution/server/lib/detection_engine/scripts/delete_all_rule_execution_data.sh

```diff
@@ -10,15 +10,31 @@
 set -e
 ./check_env_variables.sh
 
-# Example: ./delete_all_statuses.sh
+# Deletes all data related to rule execution:
+# - `siem-detection-engine-rule-execution-info` sidecar saved objects from `.kibana` index
+# - fully clears `.kibana-event-log` index
+
+# Example: ./delete_all_rule_execution_data.sh
```

Comment:
Thanks for updating the scripts! They often get missed and I know some folks are still using them, so appreciate it! 🙂